### PR TITLE
Continued Course Activity Resolvable

### PR DESCRIPTION
**Pull request by MrWilhelm**
@ragesoss Here is the one for ContinuedCourseActivityAlert

### CODE REVIEW

**Comment by ragesoss** [comment]
Lovely! Code looks great. Assuming tests pass, I'll merge it shortly.

File: spec/models/alert_spec.rb [hunk 1]
```diff
@@ -59,9 +59,24 @@
       end
     end
 
-    it 'should be resolvable only for ArticlesForDeletionAlert' do
-      expect(alert.resolvable?).to be(true)
-      expect(active_course_alert.resolvable?).to be(false)
+    it 'should be resolvable for resolvable alert types' do
+      Alert::RESOLVABLE_ALERT_TYPES.each do |type|
```
Comment (ragesoss) [inline spec/models/alert_spec.rb]: This is nice.

**Comment by MrWilhelm** [comment]
Do you have any suggestion for the next issue I will work on? @ragesoss

**Comment by ragesoss** [comment]
@MrWilhelm This one might be a good fit for you, on the strictly Ruby side: #1141

I can provide additional details on how to get necessary data. I'll start with a few pointers on that issue, and you can give me a ping if I don't say enough.

Another option, if you're up for some javascript/css, is #234 

And since you seem to know your way around Rails quite well, this one might be pretty easy for you: #1164 

I've never implemented search before, so I'm not really sure where to start.

**Comment by ragesoss** [comment]
@MrWilhelm one more option: you could do a code review of either of the larger work-in-progress PRs I have open: #1167 and #1152

Since you seem very comfortable with Rails conventions, you'd probably be able to spot ways to improve the code.

**Comment by MrWilhelm** [comment]
I think I want to get #1164 . 

I'll also check out #1167 If there is anything I can help.

I'll usually go with backend stuff since I feel more comfortable with it and easier for me to get into.

@ragesoss

**Comment by ragesoss** [comment]
@MrWilhelm great! Ping in those places if you have any questions.

File: spec/models/alert_spec.rb (hunk 1 above)
Comment (MrWilhelm) [inline spec/models/alert_spec.rb]: Thanks!

**Pull request merged**